### PR TITLE
add bottomDeleteManager

### PR DESCRIPTION
**Pull request by kibibites**
to prevent shit like this happening
Screen Shot 2021-04-12 at 11 02 18 AM

### CODE REVIEW

**Comment by 5HT2** [comment]
LOL

**Review by 5HT2 — CHANGES_REQUESTED** [review]
- Managers aren't automatically registered, you have to edit KamiBluePlugin.kt

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt [hunk 1]
```diff
@@ -19,6 +19,7 @@ class KamiBluePlugin : Plugin() {
         managers.add(PingDeleteManager)
         managers.add(PluginDownloadManager)
         managers.add(ReactionLoggingManager)
+        managers.add(BottomDeleteManager)
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt]: not alphabetical moment

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt (hunk 1 above)
Comment (syrenaaa) [reply kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt]: BRUH

**Comment by 5HT2** [comment]
Added to the responses json instead

**Pull request closed without merging**